### PR TITLE
MLOps Release Beta 1

### PR DESCRIPTION
**Pull request by CGuichard**
Pin versions for Release Beta 01

- SharingHub: "0.3.0"
- MLflow SharingHub: "0.2.0"

### CODE REVIEW

**Review by matthieurobert — APPROVED** [review]

**Review by mrobert-cs — APPROVED** [review]

File: argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml [hunk 1]
```diff
@@ -11,13 +11,13 @@ spec:
...
     path: deploy/helm/mlflow-sharinghub
-    targetRevision: main
+    targetRevision: "0.2.0"
     helm:
       valuesObject:
         image:
           repository: eoepca/mlflow-sharinghub
           pullPolicy: Always
-          tag: latest
+          tag: "0.2.0"
```
Comment (rconway) [inline argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml]: It is better if the version of the image is included (as the default) within the versioned helm chart - so it doesn't need to be specified by the user of the helm chart.

File: argocd/eoepca/mlops/parts/app-sharinghub.yaml [hunk 2]
```diff
@@ -131,7 +131,7 @@ spec:
           repository: eoepca/sharinghub
           pullPolicy: Always
           # Overrides the image tag whose default is the chart appVersion.
-          tag: "latest"
+          tag: "0.3.0"
```
Comment (rconway) [inline argocd/eoepca/mlops/parts/app-sharinghub.yaml]: Same comment as above

**Review by rconway — CHANGES_REQUESTED** [review]
I made an observation that you may want to address

File: argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml (hunk 1 above)
Comment (rconway) [reply argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml]: Typically the Chart `appVersion` is used to specify the (default) container image - https://github.com/csgroup-oss/mlflow-sharinghub/blob/main/deploy/helm/mlflow-sharinghub/Chart.yaml#L24.

BTW. the chart version here does not reflect the `0.2.0` version - https://github.com/csgroup-oss/mlflow-sharinghub/blob/main/deploy/helm/mlflow-sharinghub/Chart.yaml#L18

Comment (CGuichard) [reply argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml]: I updated the charts to use "0.2.0" by default.

File: argocd/eoepca/mlops/parts/app-sharinghub.yaml (hunk 2 above)
Comment (CGuichard) [reply argocd/eoepca/mlops/parts/app-sharinghub.yaml]: I updated the charts to use "0.3.0" by default.

**Review by rconway — APPROVED** [review]

**Pull request merged**